awx/main/models/events.py

```diff
@@ -518,7 +518,7 @@ def _update_host_summary_from_stats(self, hostnames):
             summaries = dict()
             updated_hosts_list = list()
             for host in hostnames:
-                updated_hosts_list.append(host)
+                updated_hosts_list.append(host.lower())
```

Comment:
This changes the host metrics table to fold mixed-case hosts into a single host. cc @amolgautam25

Reply:
Got it. 👍🏻